project/AppDependencies.scala

```diff
@@ -7,22 +7,24 @@ object AppDependencies {
   lazy val dependencyOverrides = overrides
 
   private lazy val bootstrapVersion = "5.16.0"
-  private lazy val hmrcReactiveMongoTestVersion = "5.0.0-play-28"
+  private lazy val hmrcMongoTestVersion = "0.64.0"
+  private lazy val hmrcMongoVersion = "0.64.0"
   private lazy val scalaJVersion = "2.4.2"
   private lazy val scalatestPlusPlayVersion = "4.0.0"
   // we need to override the akka version for now as newer versions are not compatible with reactivemongo
   private lazy val akkaVersion = "2.6.14"
```

Comment:
Is this necessary given it was added to overcome an incompatibility with reactivemongo